omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -13,18 +8,94 @@ Nginx Gunicorn configuration (Unix/Linux)
       see :ref:`python-requirements` on the
       :doc:`/sysadmins/version-requirements` page.
 
-Install `Django 1.8`_ and `Gunicorn <http://docs.gunicorn.org/>`_ using
-package requirements file:
+
+OMERO.web uses
+`Web Server Gateway Interface (WSGI) <http://wsgi.readthedocs.org/en/latest/learn.html>`_
+as a primary deployment platform, the Python standard
+`PEP 3333 <https://www.python.org/dev/peps/pep-3333/>`_
+that describes how a web server communicates with web applications.
+OMERO.web deployment can be configured with sync and async workers.
+Sync workers are faster and recommended for a data repository with
+:ref:`download_restrictions`. If you wish to offer users the ability
+to download data you have to use async workers, read more about
+:ref:`download_limitation`.
+
+
+.. _gunicorn_default_configuration:
+
+Gunicorn default configuration (Unix/Linux)
+-------------------------------------------
+
+Install `Django 1.8`_,
+`Gunicorn >= 19.3 <http://gunicorn.org>`_
+and `futures <https://pypi.python.org/pypi/futures>`_ using package requirements file:
 
 ::
 
    $ pip install -r share/web/requirements-py27-nginx.txt
 
+.. note:: For more details refer to
+      :djangodoc:`how to install Django 1.8 <topics/install/#install-the-django-code>`
+      or :djangodoc:`upgrade Django to 1.8 <topics/install/#remove-any-old-versions-of-django>`.
+
+
+Additional settings can be configured by the following properties:
```

Comment:
The comments for the previous section apply here and below:

"by changing the" for this sentence, and "to ..." for the properties lacking "to" after the property name.